### ISSUE TITLE
Optionally show number of comments in post subtitle

### ISSUE THREAD

**Issue by codeofdusk** [QuantumBadger/RedReader#989]
Per [this thread](https://www.reddit.com/r/RedReader/comments/tblskr/request_feature_post_subtitle_comment_count/#content), it'd be useful to show the number of comments on a post in the subtitle. This should be added before score in the accessibility description.

**Comment by codeofdusk** [comment]
CC @QuantumBadger.
